src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/jobs/JenkinsJobPermissionsUtils.java

```diff
@@ -35,7 +45,9 @@ private static void removePermissionsFromElement(String elementTagName, Document
         permissionsToRemove.forEach(jenkinsJobPermission -> userLogins.forEach(userLogin -> {
             // The permission in the xml node has the format: com.jenkins.job.permission:user-login
             String permission = jenkinsJobPermission.getName() + ":" + userLogin;
+            // old jobs might still use the permission without the prefix
             removePermission(authorizationMatrixElement, permission);
+            removePermission(authorizationMatrixElement, USER_PERMISSIONS_PREFIX + permission);
```

Comment:
Should we maybe include this inside the method removePermission, since other developers might use that method later on without knowing that they have to add that prefix?

Reply:
:+1: Moved in 71b7c3c6888e26ab1394f0eadc82996279a19a98